Sources/XCHammer/XcodeTarget.swift

```diff
@@ -531,6 +531,33 @@ public class XcodeTarget: Hashable, Equatable {
                 .map { "-D\($0)" }
         }
 
+        func extractSwiftVersion() -> String? {
+            // First respect swift language version if explicitly defined
+            // This should be less common and i believe was only supported by old rules_apple rules
```

Comment:
I think this part of the comment is inaccurate.

`swift_language_version` comes from `SwiftInfo.swift_version` which rules_swift populates based on the `-swift-version` flag.